test/lib/jdk/test/lib/thread/TestThreadFactory.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should just be 2024.

Reply:
I copied file from JDIThreadFactory,java (I mention this in PR) and I left copyrights from original. I think it is the correct way to treat copyrights. Not sure, if I can find policy about this though.
If you think that new file require current year only, I am fine to update it.

Reply:
That's fine I think.